### PR TITLE
Add FeatureForms for WinUI and UWP

### PR DESCRIPTION
**Pull request by dotMorten**
image

### CODE REVIEW

File: src/Toolkit/Toolkit.WinUI/UI/Controls/FeatureForm/FeatureForm.Theme.xaml [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+                    <primitives:DateTimePickerFormInputView Element="{Binding}" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="primitives:FieldFormElementView">
+                    <StackPanel>
+                        <TextBlock Text="{Binding Label}" Style="{StaticResource FeatureFormViewTitleStyle}"/>
+                        <ContentControl Margin="0,3" Foreground="Gray" Content="{Binding}" x:Name="FieldInput" IsTabStop="False" ToolTipService.ToolTip="{Binding Hint}" HorizontalContentAlignment="Stretch" />
```
Comment (mstefarov) [inline src/Toolkit/Toolkit.WinUI/UI/Controls/FeatureForm/FeatureForm.Theme.xaml:226]: I am seeing small empty tooltips pop up in the sample form:

2024-08-02_000115 Toolkit SampleApp WinUI

Is the text missing, or is this a case of an empty-string / all-whitespace hint that should be hidden?

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormInputView.cs [hunk 2]
```diff
@@ -41,14 +56,20 @@ public TextFormInputView()
 
         private void ConfigureTextBox()
         {
+#if WINDOWS_XAML
+            VisualStateManager.GoToState(this, Element?.IsEditable == false ? "Disabled" : "Enabled", true);
+            VisualStateManager.GoToState(this, Element?.Input is BarcodeScannerFormInput && Element.IsEditable ? "ShowBarcode" : "HideBarcode", true);
+            VisualStateManager.GoToState(this, Element?.IsEditable == true && Element?.Input is TextAreaFormInput ? "MultiLineText" : "SingleLineText", true);
```
Comment (mstefarov) [inline src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormInputView.cs:62]: What about non-editable "TextArea" inputs?  Shouldn't they be multi-line as well?  For example, in the sample, try writing a long description and see how "Place Summary" (read-only TextArea) is getting cut off -- no wrapping and no scrolling:

2024-08-02_000456 Toolkit SampleApp WinUI

This is different from AGOL, which has both wrapping and scrollbars:

2024-08-02_000728 msedge

File: src/Samples/Toolkit.SampleApp.UWP/Samples/Forms/FeatureFormViewSample.xaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+        <esri:MapView x:Name="mapView"
+               Map="{x:Bind Map}"
+               GeoViewTapped="mapView_GeoViewTapped"/>
+
+        <Border HorizontalAlignment="Center" VerticalAlignment="Top" Background="#99ffffff" Padding="5" Margin="10" BorderThickness="1" BorderBrush="Black">
+            <TextBlock Text="Click a feature to open its FeatureFormView" FontSize="18" />
```
Comment (mstefarov) [inline src/Samples/Toolkit.SampleApp.UWP/Samples/Forms/FeatureFormViewSample.xaml:21]: Sample: This text is unreadable (appears white-on-white) when using dark mode.

2024-08-02_001613 Toolkit SampleApp WinUI

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/RadioButtonsFormInputView.cs [hunk 4]
```diff
@@ -115,7 +130,7 @@ private void UpdateSelection()
                 var selection = input.CodedValues.Where(a => object.Equals(a.Code, Element?.Value)).FirstOrDefault();
                 if (selection is null && input.NoValueOption == FormInputNoValueOption.Show)
                 {
-#if MAUI
+#if MAUI || WINDOWS_XAML
                     SelectedItem = null;
```
Comment (mstefarov) [inline src/Toolkit/Toolkit/UI/Controls/FeatureForm/RadioButtonsFormInputView.cs:134]: Selecting the "no value" radio button doesn't seem to work on first click -- it becomes unchecked.
2024-08-02_005218 Toolkit SampleApp WinUI

This is different from WPF, where "no value" radio button options works on the same map.

**Review by mstefarov — APPROVED** [review]
Looks good!

**Review by prathameshnarkhede — APPROVED** [review]

**Pull request merged**